fib.c

```diff
@@ -0,0 +1,24 @@
+/* Matthew Danielson
+*  7/5/16
+*  fib.c
+*  Gets the sum of the fibonacci sequence up to a certain point
+*/
+
+#include <stdio.h>
+
+int main(){
+	printf("Up to what number of the Fibonacci sequence?");
+	int max;
+	scanf("%d", &max);
+	int array[max+2];
+	array[0]=1;
```

Comment:
you took care of the base cases, but didn't have them printed out, so the code actually printed out the wrong fibonacci numbers.